charts/kubebb-core/templates/pipeline.yaml

```diff
@@ -0,0 +1,136 @@
+{{- if .Values.deployment.rating_enable }}
+apiVersion: tekton.dev/v1beta1
```

Comment:
@0xff-dev Can we also check if tekton crds exists? 

Like `{{ if .Capabilities.APIVersions.Has "tekton.dev/v1beta1" -}}`